crates/matrix-sdk-ui/src/sync_service.rs

```diff
@@ -315,6 +315,47 @@ struct SyncServiceInner {
     supervisor: Option<SyncTaskSupervisor>,
 }
 
+/// A high level manager for your Matrix syncing needs.
+///
+/// The [`SyncService`] is responsible for managing real-time synchronization
+/// with a Matrix server. It can initiate and maintain the necessary
+/// synchronization tasks for you.
```

Comment:
Thanks! That's exactly what we often miss to do in the SDK, this kind of doc is really useful.